src/Data/Gamedata.ts

```diff
@@ -20,6 +20,7 @@ export const Fortunes: string[] = [
   'Fortune #016',
   'Fortune #017',
   'Fortune #018',
+  'Fortune #019',
```

Comment:
I don't see a new Fortune `#019` in http://orteil.dashnet.org/cookieclicker/main.js?v=2.099
Am I missing something?

Reply:
Oh weird... My `ctrl + f` is not good enough it seems 😄